examples/warm_up_example/warmed_up_fedprox/server.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+from functools import partial
+from logging import INFO
+from typing import Any, Dict, Optional
+
+import flwr as fl
+from flwr.common.logger import log
+from flwr.common.parameter import ndarrays_to_parameters
+from flwr.common.typing import Config, Parameters
+from flwr.server.client_manager import SimpleClientManager
+
+from examples.models.cnn_model import MnistNet
+from examples.simple_metric_aggregation import evaluate_metrics_aggregation_fn, fit_metrics_aggregation_fn
+from examples.utils.functions import make_dict_with_epochs_or_steps
+from fl4health.reporting.fl_wanb import ServerWandBReporter
+from fl4health.server.base_server import FlServer
+from fl4health.strategies.fedprox import FedProx
+from fl4health.utils.config import load_config
+from fl4health.utils.random import set_all_random_seeds
+
+
+def get_initial_model_information() -> Parameters:
+    # Initializing the model parameters on the server side.
+    # Currently uses the Pytorch default initialization for the model parameters.
+    initial_model = MnistNet()
+    model_weights = [val.cpu().numpy() for _, val in initial_model.state_dict().items()]
+    return ndarrays_to_parameters(model_weights)
```

Comment:
To make it clear that we're not using the server-side initial parameters, perhaps we can simply do something like
```
def get_initial_model_information() -> Parameters:
    # Server-side initial weights are not used for client side models when using a warmed up module. So these can be initialized as empty
    return ndarrays_to_parameters([])
```

Note: I think this will work, but I haven't tested it. 😂